Unable to complete agent check-in to CCM

As per #85 which installs `chocolatey-agent` on the server, and then checks in to CCM, the C4B Server is not checking into itself.

The agent log states the following:
```
2021-10-22 14:45:41,972 6 [ERROR] - Unable to set up central management reporting and checking for
 deployments. It appears you've enabled the feature
 'useChocolateyCentralManagement', but you have not defined the
 management service url. Please ensure you set the configuration
 value centralManagementServiceUrl (e.g. 'choco config
 set centralManagementServiceUrl
 https://<location>:<port>/ChocolateyManagementService'). For more
 information, please see
 https://chocolatey.org/docs/features-chocolatey-central-management.
```

Again, @steviecoaster, assigning this you Sir. Please investigate.

--- a/scripts/Get-Helpers.ps1
+++ b/scripts/Get-Helpers.ps1
@@ -1711,7 +1711,7 @@ function Install-ChocolateyAgent {
         }
         
 
-        $chocoArgs = @('config','set','centralMangementServiceUrl',"$CentralManagementServiceUrl")
+        $chocoArgs = @('config','set','centralManagementServiceUrl',"$CentralManagementServiceUrl")
         & choco @chocoArgs
 
         $chocoArgs = @('feature','enable','--name="useChocolateyCentralManagement"')

